index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
instead of displaying refresh to play again , it would be more better to Are you ready for the next game.. Click here

Reply:
> instead of displaying refresh to play again , it would be more better to Are you ready for the next game.. Click here

```suggestion
<h2 hidden>Game over, Are you ready for the next game..<a href = "/index.html"> Click here </a>🧑‍💻 🤖!</h2>
```